source/Applications/Advanced/HandEyeCalibration/PoseConversions/PoseConversions.cpp

```diff
@@ -66,12 +61,32 @@ int main()
         std::cout << "RotationMatrix:\n" << transformationMatrix.linear() << std::endl;
         std::cout << "TranslationVector:\n" << transformationMatrix.translation() << std::endl;
 
-        // Convert from Zivid to Robot (Transformation Matrix --> any format)
-        const auto robotRotationRepresentations = zividToRobot(transformationMatrix);
-
-        // Convert from Robot to Zivid (any format --> Rotation Matrix)
-        const auto transformationMatrix2 =
-            robotToZivid(robotRotationRepresentations, transformationMatrix.translation());
+        /*
+         * Convert from Zivid to Robot (Transformation Matrix --> any format)
+         */
+        print_header("Convert from Zivid (Rotation Matrix) to Robot");
+        const Eigen::AngleAxisd axisAngle(transformationMatrix.linear());
+        std::cout << "AxisAngle:\n" << axisAngle.axis() << ", " << axisAngle.angle() << std::endl;
```

Comment:
Is the output supposed to look like this? (I don't know the domain, so might very well be)
```
AxisAngle:
-0.9556
 0.2724
 0.1127, 0.275
```

Reply:
Yeah, that one is a bit annoying. For Python it renders as:
```python
AxisAngle:
[-0.9556  0.2724  0.1127], 0.2750
```
I'll push a commit that makes it look the same.